packages/core-js/utils/ast.js

```diff
@@ -92,7 +96,45 @@ function findFunctionSelectors(contractName, asts) {
   return selectors;
 }
 
+// prettier-ignore
+function orderContractsStructMap(structs) {
+  structs.sort((a, b) =>
+    a.contract.name > b.contract.name
+      ? 1
+      : a.contract.name < b.contract.name
+        ? -1
+        : a.struct.name > b.struct.name
+          ? 1
+          : a.struct.name < b.struct.name
+            ? -1
+            : 0
```

Comment:
Here looks like that is doing several times the same comparison, e.g.:  it does 2 times `a.contract.name > b.contract.name`

Reply:
is checking **contract.**name before ( `>` and `<` ) and later **struct.**name